WooCommerce/Classes/Authentication/Epilogue/EmptyStoresTableViewCell.xib

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="21179.7" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="21507" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
```

Comment:
Super nit: There seems to be a background color difference when dark mode is turned on. 
<img src="https://user-images.githubusercontent.com/524475/202323462-b53684f1-8807-4c7c-b84b-9d0df99387b5.png" width="300"/>

Reply:
Awesome, I missed this! Fixed in d14063de27a0b350786e38a39fe4c9f3c1dd06af.